src/hdf5_interface.F90

```diff
@@ -787,6 +791,76 @@ subroutine hdf5_read_string(group, name, buffer, length)
   end subroutine hdf5_read_string
 
 !===============================================================================
+! HDF5_WRITE_STRING_1DARRAY writes string data
+!===============================================================================
+
+  subroutine hdf5_write_string_1Darray(group, name, buffer, length, rank)
```

Comment:
You're right, this routine was used at an earlier stage in this PR when I was writing out all of the labels for the tallies as an array, and then linking them back to the appropriate tallies in the StatePoint Python API. Now I am writing out each label separately within each tallies HDF5 group, so this routine is no longer needed. I just removed it with my latest commit.